### DISCUSSION TITLE
Color parameter list.

### DISCUSSION THREAD

**Discussion by Pieter3033** [sqlpage/SQLPage#106] [Q&A] [answered]
Hi Lovasoa.

Is there a reference list for all the available colors?
I'm guessing all the components color parameters are the same.

Thank you kindly.

**Comment by lovasoa** [comment, marked as answer]
Hello ! 
  I will improve the documentation to make this clearer. The list of color names is here: https://tabler.io/docs/base/colors

**Reply by Pieter3033** [reply]
Thank you sir.

**Comment by lovasoa** [comment]
I'm adding links to `color` and `icon` properties in  the documentation to allow quick access to the reference.
